ui/lib/sync/addon/components/secrets/page/destinations.hbs

```diff
@@ -12,4 +12,114 @@
       <LinkTo @route="secrets.destinations" data-test-tab="Destinations">Destinations</LinkTo>
     </ul>
   </nav>
-</div>
\ No newline at end of file
+</div>
+
+<Toolbar>
+  <ToolbarFilters>
+    <SearchSelect
+      @options={{this.destinationTypes}}
+      @objectKeys={{array "id" "name"}}
+      @passObject={{true}}
+      @selectLimit={{1}}
+      @disallowNewItems={{true}}
+      @placeholder="Filter by type"
+      @inputValue={{if this.typeFilter (array this.typeFilter)}}
+      @onChange={{fn this.onFilterChange "type"}}
+      class="is-marginless"
```

Comment:
this class might cover this, but I've noticed that for search selects in toolbars we usually pass `displayInherit = true`

Reply:
I noticed that as well! It was originally on the component I copied from somewhere else and when I removed it I didn't notice any visual changes.

Reply:
Okay cool!